src/main/java/com/celements/search/web/classes/WebSearchClassPackage.java

```diff
@@ -12,7 +12,7 @@
 @Component(WebSearchClassPackage.NAME)
 public class WebSearchClassPackage extends AbstractClassPackage {
 
-  public static final String NAME = "webSearch";
+  public static final String NAME = "websearch";
```

Comment:
package and collection is not the same and are configured in different places:
- WebSearchClasses (deprecated) implements IClassCollectionRole, configured in _xwiki.cfg_. I haven't changed their name "webSearch"
- WebSearchClassPackage (new) implements ClassPackage, configured in ConfigurationSource. This package has not yet been released.